pkg/capture/capture_manager.go

```diff
@@ -454,6 +459,30 @@ func (cm *Manager) rotate(ctx context.Context, writeoutChan chan<- capturetypes.
 	).Debug("rotated interfaces")
 }
 
+func (cm *Manager) logErrors(ctx context.Context, iface string, errsChan <-chan error) {
+	logger := logging.FromContext(ctx)
+	for {
+		select {
```

Comment:
If the capture was terminated in a "regular" fashion it should have happened via the context. Otherwise it has happened because of the error channel being closed from within `process()` first, which would indicate a critical error and trigger teardown of the interface.